src/extension/remoteConfigHelper.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as https from "https";
+import { CancellationTokenSource } from "vscode";
+import { PromiseUtil } from "../common/node/promise";
+
+export interface IConfig {}
+
+export function downloadConfig<T extends IConfig | IConfig[]>(endpointURL: string): Promise<T> {
+    return new Promise<T>((resolve, reject) => {
+        https
+            .get(endpointURL, response => {
+                let data = "";
+                response.setEncoding("utf8");
+                response.on("data", (chunk: string) => (data += chunk));
+                response.on("end", () => {
+                    try {
+                        resolve(JSON.parse(data));
+                    } catch (err) {
+                        reject(err);
+                    }
+                });
+                response.on("error", reject);
+            })
+            .on("error", reject);
+    });
```

Comment:
There is already implemented the same [function](https://github.com/microsoft/vscode-react-native/blob/master/src/common/node/request.ts#L7)